metaparser/morph/mrd.lua

```diff
@@ -787,7 +787,11 @@ function mrd:create(fname, crc)
 		return
 	end
 	for _, d in ipairs(self.dirs) do
-		for f in std.readdir(instead.gamepath() .. '/'..d) do
+    path = '.'
+    if instead.gamepath ~= nil then
+      path = instead.gamepath()
+    end
+		for f in std.readdir(path .. '/'..d) do
```

Comment:
We are using tabs, not spaces to indent....
Anyway, can you describe this change? What it solves?

Reply:
a) saves `instead.gamepath` to variable, so we're not calling `gamepath()` function in a loop (`readdir` is also called there, btw)
b) if there is no `instead.gamepath` function (i.e. we're running tiny-instead), just use current dir